pymongo/synchronous/periodic_executor.py

```diff
@@ -28,6 +28,99 @@
 _IS_SYNC = True
 
 
+class SyncPeriodicExecutor:
```

Comment:
We should organize the code such that we don't sync this code since it will never be used.

Reply:
As in move this file out of the async / sync folders right? If so, that makes sense to me :)

Reply:
Yep. We can have one file that has both.